src/main/resources/test_files/runtime-spark/NestedFLWORSparkError1.iq

```diff
@@ -0,0 +1,5 @@
+(:JIQS: ShouldCrash; ErrorCode="RBS0003"; ErrorMetadata="LINE:2:COLUMN:0:" :)
```

Comment:
Here it says RBS0003, but the error is RBST0003 -- it would be nice to double-check that there is no bug in the test machinery, do make sure that this is not a false positive?